CHANGELOG.md

```diff
@@ -2,6 +2,30 @@
 
 ## [Unreleased]
 
+## [0.19.0] - 2019-09-26
+### Added
+- Added `--workspace-dir` option to allow specifying custom workspace directory
+- Added **sonamecheck** checker for detecting *SONAME* changes
+- Added `--copr-project-permanent`, `--copr-project-frontpage` and `--copr-chroots` options
+
+### Fixed
+- Strings like "1" are no longer replaced with macros in *%prep*
+- Fixed broken `--comparepkgs-only` option
```

Comment:
This line should be removed, I eventually removed the option as it turned out to still be problematic in most cases.

Reply:
Good catch, thanks.